config/class-sync.php

```diff
@@ -123,6 +124,17 @@ public function sync_jetpack_privacy_settings( $expected_value, $value_from_opti
 		return $success;
 	}
 
+	public function put_site_details() {
+		// This is a temporary workaround while the full Site Details implementation is finished
+		// In order to reduce the amount of stored documents, we increase the execution interval
+		// from 30 minutes to approximately 4 hours
+		if ( 1 === rand( 1, 8 ) ) {
+			require_once( 'class-site-details-index.php' );
```

Comment:
```suggestion
			require_once( __DIR__ .  '/class-site-details-index.php' );
```

In general it's more robust to use an absolute path (and it's what we do in most other spots in mu-plugins).

Reply:
Sure! Changed that